kili/cli/cli.py

```diff
@@ -0,0 +1,47 @@
+"""Kili CLI"""
+import click
+from kili.cli.common_args import CONTEXT_SETTINGS
+
+from kili import __version__
+
+
+from kili.cli.project.create import create_project as create
+from kili.cli.project.describe import describe_project as describe
+from kili.cli.project.import_asset import import_assets
+from kili.cli.project.import_label import import_labels
+from kili.cli.project.list import list_project
+
+
+@click.group(context_settings=CONTEXT_SETTINGS)
+@click.version_option(__version__)
+def cli():
+    """Kili Command line Interface
+
+       To get all the available commands, please type: `kili project --help`.
+    """
+
+
+@cli.group(context_settings=CONTEXT_SETTINGS)
+def project():
+    """Commands to interact with a Kili project"""
+
+
+project.add_command(create, name="create")
+project.add_command(describe, name="describe")
+project.add_command(import_assets, name="import")
```

Comment:
why not renaming the python functions into the command names? this would be even clearer.

Reply:
You mean : `from kili.cli.project.create import create_project` then `project.add_command(create_project, name="create")`. If so, then I agree.

Reply:
```
from kili.cli import project
project.add_command(project.create, name="create")
```